Pages/Register.cshtml

```diff
@@ -1,44 +1,139 @@
 ﻿@page
 @model FoodKept.Pages.RegisterModel
 @{
+    ViewData["Title"] = "Register";
 }
 
 <div class="container mt-5">
     <div class="row justify-content-center align-items-center">
         <div class="col-sm-12 col-md-12 col-lg-4">
             <h1 class="mb-3">Register</h1>
+            <ul class="nav nav-tabs">
```

Comment:
In my opinion, this html should be divided in 2 files. One for customers register, second - restaurant register.
Because now it is little bit hard to read.